docs/user-signature.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: Signing and Verifying Arbitrary Data
+---
+
+## Signing Arbitrary Data
+
+Cryptographic signatures are a key part of the blockchain. They are used to prove ownership of an address without exposing its private key.
+This is primarily used for signing transactions but can also be used to sign arbitrary messages.
```

Comment:
This can't be used to sign transactions :P I know what you meant here but I think we might need to make it a little bit clearer?

Reply:
How about this phrasing?

_While primarily used for signing transactions, cryptographic signatures can also be used to sign arbitrary messages._